user-guide/toc.yml

```diff
@@ -1945,6 +1934,80 @@
                 topicUid: DashboardNumericInputFeed
               - name: Text input
                 topicUid: DashboardTextInputFeed
+        - name: Generic Query Interface (GQI)
+          topicUid: Generic_Query_Interface
+          items:
+            - name: Creating a GQI query
+              topicUid: Creating_GQI_query
+            - name: Query data sources
+              topicUid: Query_data_sources
+            - name: Query operators
+              topicUid: Query_operators
+              items:
```

Comment:
It would be nice if we could also list the Custom Operator in here as a separate page, similar to how the Ad Hoc data source is listed in "Query Data Source".
That way the user can find it both by looking in Extension and looking through the operators. 
It can just link to the page in Extensions.

Reply:
Implemented in [commit 634dceb](https://github.com/SkylineCommunications/dataminer-docs/pull/1694/commits/634dceb95d27f7c4d63b43ef9ab1a88eb3603209).